media/transcoder/create_job_with_periodic_images_spritesheet.go

```diff
@@ -0,0 +1,117 @@
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/golang/protobuf/ptypes/duration"
+
+	transcoder "cloud.google.com/go/video/transcoder/apiv1beta1"
+	transcoderpb "google.golang.org/genproto/googleapis/cloud/video/transcoder/v1beta1"
+)
+
+// createJobWithPeriodicImagesSpritesheet creates a job from an ad-hoc configuration and generates two spritesheets from the input
+// video. Each spritesheet contains images that are captured periodically based on a user-defined time interval.
+func createJobWithPeriodicImagesSpritesheet(w io.Writer, projectID string, location string, inputURI string, outputURI string) error {
+	// projectID := "my-project-id"
+	// location := "us-central1"
+	// inputURI := "gs://my-bucket/my-video-file"
+	// outputURI := "gs://my-bucket/my-output-folder/"
+	ctx := context.Background()
+	client, err := transcoder.NewClient(ctx)
+	if err != nil {
+		return fmt.Errorf("NewClient: %v", err)
+	}
+	defer client.Close()
+
+	req := &transcoderpb.CreateJobRequest{
+		Parent: fmt.Sprintf("projects/%s/locations/%s", projectID, location),
+		Job: &transcoderpb.Job{
+			InputUri:  inputURI,
+			OutputUri: outputURI,
+			JobConfig: &transcoderpb.Job_Config{
+				Config: &transcoderpb.JobConfig{
+					ElementaryStreams: []*transcoderpb.ElementaryStream{
+						&transcoderpb.ElementaryStream{
+							Key: "video_stream0",
+							ElementaryStream: &transcoderpb.ElementaryStream_VideoStream{
+								VideoStream: &transcoderpb.VideoStream{
+									Codec:        "h264",
+									BitrateBps:   550000,
+									FrameRate:    60,
+									HeightPixels: 360,
+									WidthPixels:  640,
+								},
+							},
+						},
+						&transcoderpb.ElementaryStream{
+							Key: "audio_stream0",
+							ElementaryStream: &transcoderpb.ElementaryStream_AudioStream{
+								AudioStream: &transcoderpb.AudioStream{
+									Codec:      "aac",
+									BitrateBps: 64000,
+								},
+							},
+						},
+					},
+					MuxStreams: []*transcoderpb.MuxStream{
+						&transcoderpb.MuxStream{
+							Key:               "sd",
+							Container:         "mp4",
+							ElementaryStreams: []string{"video_stream0", "audio_stream0"},
+						},
+					},
+					SpriteSheets: []*transcoderpb.SpriteSheet{
+						&transcoderpb.SpriteSheet{
+							FilePrefix:         "small-sprite-sheet",
+							SpriteWidthPixels:  64,
+							SpriteHeightPixels: 32,
+							ExtractionStrategy: &transcoderpb.SpriteSheet_Interval{
+								Interval: &duration.Duration{
+									Seconds: 7,
+								},
+							},
+						},
+						&transcoderpb.SpriteSheet{
+							FilePrefix:         "large-sprite-sheet",
+							SpriteWidthPixels:  128,
+							SpriteHeightPixels: 72,
+							ExtractionStrategy: &transcoderpb.SpriteSheet_Interval{
+								Interval: &duration.Duration{
+									Seconds: 7,
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+	// Creates the job, Jobs take a variable amount of time to run.
+	// You can query for the job state.
```

Comment:
Say how to query? Often, other samples show waiting for the job to finish directly in the sample.

Reply:
Updated the comment.